### PR TITLE
Add version notes to Drupal docs

### LINKED ISSUE #3011: Rework SimpleSAML page
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/guides/drupal9/simplesaml.html

### What exactly should be updated?

We got feedback that now that 2.0.x versions of SimpleSAML are available, our dedicated doc page is outdated (caters to 1.x versions). Let's rework and update it.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Closes #3011 and #3237

## What's changed

Added version notes to Drupal docs (& specific one to SimpleSAML page).

### CODE REVIEW

File: docs/src/guides/drupal9/_index.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ weight: -180
 sectionBefore: PHP
 description: |
     Everything you need to get started with Drupal on Platform.sh.
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (chadwcarlson) [inline docs/src/guides/drupal9/_index.md:9]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

**Review by chadwcarlson — CHANGES_REQUESTED** [review]
Needs to be the following for all

File: docs/src/guides/drupal9/deploy/_index.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ weight: -110
 layout: single
 description: |
     Create a Platform.sh account, download a few tools, and prepare to deploy Drupal.
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (AnouckColson) [inline docs/src/guides/drupal9/deploy/_index.md:10]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

File: docs/src/guides/drupal9/deploy/configure.md [hunk 3]
```diff
@@ -4,6 +4,9 @@ sidebarTitle: "Configure"
 weight: -100
 description: |
     Review the basics of what makes up a Platform.sh project, including its three principle configuration files and how to define them for Drupal.
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (AnouckColson) [inline docs/src/guides/drupal9/deploy/configure.md:9]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

File: docs/src/guides/drupal9/deploy/customize.md [hunk 4]
```diff
@@ -4,6 +4,9 @@ sidebarTitle: "Customize"
 weight: -90
 description: |
     Add some helpful dependencies, and modify your Drupal site to read from a Platform.sh environment.
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (AnouckColson) [inline docs/src/guides/drupal9/deploy/customize.md:9]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

File: docs/src/guides/drupal9/deploy/deploy.md [hunk 5]
```diff
@@ -4,6 +4,9 @@ sidebarTitle: "Deploy"
 weight: -80
 description: |
     Now that your site is ready, push it to Platform.sh and import your data.
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (AnouckColson) [inline docs/src/guides/drupal9/deploy/deploy.md:9]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

File: docs/src/guides/drupal9/deploy/next-steps.md [hunk 6]
```diff
@@ -4,6 +4,9 @@ sidebarTitle: "Next steps"
 weight: -70
 description: |
     Upgrading, adding modules, and further development of your site.
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (AnouckColson) [inline docs/src/guides/drupal9/deploy/next-steps.md:9]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

File: docs/src/guides/drupal9/multi-site.md [hunk 7]
```diff
@@ -5,6 +5,9 @@ toc: false
 description: |
     Platform.sh supports running [multiple applications](/bestpractices/oneormany.md) in the same project, and these can be two or more Drupal sites.
 weight: -80
+banner:
+    title: A note on version
+    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh/template-builder/tree/master/templates/drupal10) is available.
```
Comment (AnouckColson) [inline docs/src/guides/drupal9/multi-site.md:10]:
```suggestion
    body: While this guide focuses on Drupal 9, you can also refer to it when using Drupal 10 as differences in settings are minimal. Note that a Platform.sh [Drupal 10 template](https://github.com/platformsh-templates/drupal10) is available.
```

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**